index-pt-br.html

```diff
@@ -63,7 +63,7 @@
 </style>
 </head>
 <body>
-<h1>Código de conduta da conferência</h1>
+<h1>Código de Conduta da Conferência</h1>
```

Comment:
The official rule for titles in Portuguese is lowercase. Capitalising nouns is English. 🙂

[Reference](https://pt.wikipedia.org/wiki/Wikipédia:Votações/Regra_para_t%C3%ADtulos#Proposta_1:_Norma_da_ABNT/Regra_do_acordo_ortográfico_de_1990).

Reply:
Hi, Matt! I'm aware of that constraint about title captalisation but also there is an rule that says proper name must be capitalised noums. If we are naming the conduct code as 'Código de Conduta da Conferência' then i guess it should be Capitalised. What are your thoughts about this? Thanks!

Reply:
Matt, as this discussion is not settled yet I'll revert those changes and send only the typo fixes. It's more important change this than the title one. We can discuss the title matter later. Thanks!

Reply:
👌 
Thanks a lot for this!